homeassistant/components/somfy/cover.py

```diff
@@ -70,6 +78,15 @@ def set_cover_position(self, **kwargs):
         """Move the cover shutter to a specific position."""
         self.cover.set_position(100 - kwargs[ATTR_POSITION])
 
+    @property
+    def device_class(self):
+        """Return the device class."""
+        if self.categories & BLIND_DEVICE_CATAGORIES:
+            return DEVICE_CLASS_BLIND
+        elif self.categories & SHUTTER_DEVICE_CATAGORIES:
```

Comment:
You can simply use an if as you return in the morning block.

Reply:
Will fix.

I also realized that `self.cover.open()` and `self.cover.close()` blocks until the operation finihses

Reply:
Are you sure? So if 10s is required to open your cover we are block 10s?